frameworks/helloworld/universe/config.json

```diff
@@ -130,6 +130,11 @@
           },
           "description": "Placement constraint for the hello-world scheduler",
           "default": []
+        },
+        "verbose_mesos_logging": {
+          "description": "Sets the value of GLOG_v env-var used by Mesos. Enabled by default, set to 0 to disable.",
+          "type": "integer",
```

Comment:
Is the list of possible values exhaustive? If yes, how about making this enum?!

Reply:
Good point, pushing up patch in next commit.